.gitignore

```diff
@@ -1,4 +1,7 @@
 .idea
+
+server/tests/db.sqlite
```

Comment:
Why do we need sqlite, if we can run postgres locally?

Reply:
I agree that for local development we can use postgres, we can just use a simple docker-compose file. On the other hand, for testing it could make sense using sqlite imo.

Reply:
This is for the local unit tests, so that they can run without docker and otherwise any running services.

Reply:
yeah, I know sounds fine. the only question is how will populate this db

Reply:
I plan to dump from the staging environment once we have it and ORMize it in `sample_db_data.py`